d2bs/kolbot/libs/bots/Mephisto.js

```diff
@@ -165,11 +165,14 @@ function Mephisto() {
 
 	if (Config.OpenChests) {
 		Pather.moveTo(17572, 8011);
-		Attack.openChests(5);
+		Chest.scan(5);	//Chest stuff should no longer be necessary but leaving it here just in case
+		Chest.openChests();
```

Comment:
same comment as before regarding scan

Reply:
* Removed Chest.scan() calls
* Added parameter 5 to Chest.openChests()